content/en/blog/2024/otel-cicd-sig/index.md

```diff
@@ -0,0 +1,145 @@
+
+## CI/CD Is part of the latest OpenTelemetry Semantic Conventions
+
+Over the course of months of iteration and feedback, the [first set of Semantic Conventions was merged](https://github.com/open-telemetry/semantic-conventions/pull/1075) in for the v1.27.0 release. This change brought forth the first set of foundational semantics for CI/CD under the `CICD`, `artifacts`, `VCS`, `test`, and `deployment` namespaces. This was a significant milestone for the CI/CD Observability SIG and industry as a whole. This creates the foundation for which all of our group’s other goals can begin to take form, and reach implementation.
+
+But what does that actually mean? What value does it provide? Let’s consider real world examples for two of the namespaces
+
+### Tracking release revisions from Version Control Systems (VCS)
+
+[Version Control System (VCS) attributes](https://opentelemetry.io/docs/specs/semconv/attributes-registry/vcs/) cover multiple areas common in a VCS like refs and changes (pull/merge requests). The `vcs.repository.ref.revision` attribute is a key piece of metadata. As Version Control Systems like GitHub and GitLab emit events, they can now have this semantically compliant attribute. That means when integrating code, releasing it, and deploying it to environments, systems can include this attribute and trace the code revision across bounds more easily. In the event a deployment fails, you can quickly look at the revision of code and track it back to the buggy release. This attribute is actually a key piece of metadata for [DORA metrics](https://dora.dev/guides/dora-metrics-four-keys/) too as you calculate Change lead time and Failed deployment recovery time.
+
+### Artifacts for supply chain security, aligned with the SLSA specification
+
+The [artifact attribute namespace](https://opentelemetry.io/docs/specs/semconv/attributes-registry/artifact/) had multiple attributes for its first implementation. One key set of attributes within this namespace cover [attestations](https://slsa.dev/attestation-model) that closely align with the [SLSA](https://slsa.dev/spec/v1.0/about) model. This is really the first time a direct connection is being made between Observability and Software Supply Chain Security. Consider the following [supply chain threat model](https://slsa.dev/spec/v1.0/threats) defined by SLSA:
+![SLSA supply chain threat model diagram](SLSA-supply-chain-model.png)
```

Comment:
from what I understand this image is by SLSA? We need to add a license note to that image, please review the project's license to find out what is needed for that (it seems to be custom so I can not provide guidance):

https://github.com/slsa-framework/slsa/blob/main/LICENSE.md

depending on the order of things, we might follow https://github.com/open-telemetry/opentelemetry.io/issues/5681 already

```suggestion
{{< figure class="figure" src="SLSA-supply-chain-model.png" attr="ADD LICENSE NOTICE HERE" attrlink="https://github.com/slsa-framework/slsa/blob/a2660277e5c4ec587eeae17edacb3d199395349e/LICENSE.md" >}}
```

Reply:
thanks @adrielp. 

@horovits based on that can you update `ADD LICENSE NOTICE HER` in my suggestion above with the required attribution?